types/schemas/prototypeApplication/enums/ApplicationType.ts

```diff
@@ -587,9 +506,106 @@ type PPOutlineMajorSomeWaste = 'pp.outline.major.some.waste';
  */
 type PPPip = 'pp.pip';
 
-export type PPApplicationType =
-  | PP
-  | PPFull
+/**
+ * @description Rights of Way Order - Apply to move or close a path
+ */
+type RightsOfWayOrder = 'rightsOfWayOrder';
+
+/**
+ * @description Works to trees
+ */
+type WTT = 'wtt';
+
+/**
+ * @description Consent to carry out works to a tree with a Tree Preservation Order
+ */
+type WTTConsent = 'wtt.consent';
+
+/**
+ * @description Notification of proposed works to a tree in a Conservation Area
+ */
+type WTTNotice = 'wtt.notice';
+
+/**
+ * @id #ApplicationType
+ * @description Planning application types
+ */
+export type ApplicationType =
+  | AdvertConsent
+  | Amendment // Service descriptor, not actually submittable
```

Comment:
These `// Service descriptor, not actually submittable` comments are a little tricky ! 

We want to still include "prefixes" in overall list of enum values because these enable data visualisation like tree maps/dendrograms I think, but we don't actually ever expect a plain "ldc" to get submitted - hence example payloads will always be granular type like `ldc.existing`, etc ! 

May explore removing these later but should be harmless for now :ok_hand: